.github/workflows/nightly.yml

```diff
@@ -0,0 +1,29 @@
+name: Nightly build
+on:
+  schedule:
+    - cron: '0 2 * * *' # run at 2 AM UTC
+
+jobs:
+  linux-nightly-build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        ref: 'develop'
+    
+    - name: Install tools
+      run: sudo apt install libboost-all-dev
+        
+    - name: Configure CMake
+      run: |
+        mkdir build && cd build
+        cmake .. -DBUILD_TESTING=ON -DCMAKE_BUILD_TYPE=Release
```

Comment:
without the option "-DBUILD_CODE_COVERAGE=ON" ?
I thought that was the point of the nigthly bluid...